agent/consul/rpc_test.go

```diff
@@ -347,9 +377,9 @@ func TestRPC_GRPC(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
 	defer cancel()
 
-	stream, err := streamClient.Stream(ctx, &TestRequest{})
+	stream, err := streamClient.Subscribe(ctx, &SubscribeRequest{Key: "health/service-nodes/redis"})
```

Comment:
I feel like "health/service-nodes" should be the topic and "redis" the key. Is that the plan. One reason I suggested this simplified version for now is to just sidestep complexities like escaping delimeters that might legally exist in service names etc.